index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Raj Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+</head>
+<body>
+  <main class="rj">
+    <header>
+      <div>      <a href="#" class="logo">My Logo</a></div>
+
+      <div>
+        <img src="image/Union.png" alt="button" />
+      
+    </div>
+
+    </header>
+    <section>
+      <h1>I'm Vivek Raj glad to see you !</h1>
+      <p class="software">
+        I’m a software developer! I can help you build a product, 
+        feature or website Look through some of my work and 
+        experience! If you like what you see 
+        and have a project you need coded, 
+        don’t hestiate to contact me.
+      </p>
+      <p class="connect">LET'S CONNECT</p>
+      <div class="media">
+        <ul class="icon">
+          <li>
+            <a href="https://twitter.com/BossVivekRaj">
+              <img src="image/twitter.png" alt="twitter-icon" />
+            </a>
+          </li>
+          <li>
+            <a href="https://www.linkedin.com/in/vivek-raj-a78b92256/">
+              <img src="image/Linkedin icon.png" alt="linkedin-icon" />
+            </a>
+          </li>
+          <li>
+            <a href="https://medium.com/">
+              <img src="image/medium.png" alt="medium-icon" />
+            </a>
+          </li>
+          <li>
+            <a href="https://github.com/VivekRaaZ">
+              <img src="image/github.png" alt="github-icon" />
+            </a>
+          </li>
+          <li>
+            <a href="https://www.angellist.com/">
+              <img src="image/angellist.png" alt="angellist-icon" />
+            </a>
+          </li>
+        </ul>
+      </div>
+    </section>
```

Comment:
- [ ] You should stick to the design as much as possible like color, spacing, font, etc. I would suggest, please use the small background image as some are missing. Also place it at the proper place.
### Your design
![hisssssss](https://github.com/VivekRaaZ/portfolio/assets/81727988/8d1a94f1-658a-4a35-b5d0-5af11cb1ca04)

### Figma design
![his](https://github.com/VivekRaaZ/portfolio/assets/81727988/c6c1c33b-1928-44e1-884d-22603a59b38e)

Reply:
@roshan-bajgain 
![Screenshot 2023-07-08 141037](https://github.com/VivekRaaZ/portfolio/assets/114300476/49632c48-ffad-4382-afb0-c65b9c7478b5)

it's working very well, i think you change the right width and height then it mismatch the bubble.

Reply:
Hello @VivekRaaZ, I notice that in your section class, you have `height 78%` You can fix this issue by making it `100%` or `100vh` but you need to decrease the gap between the header and section of the page.
As it show only upto `78%` height only that's why the background image is not showing.
### You css file
![his](https://github.com/VivekRaaZ/portfolio/assets/81727988/071be348-1157-4fce-95a4-f4b1b826adb8)